api.py

```diff
@@ -31,28 +34,25 @@ def shutdown_server():
     os._exit(0)
 
 
+# DEPRECATED
 @app.route("/api/v1/url", methods=["POST"])
 def check_url():
     json = request.get_json()
 
-    # main_logger.debug("Received JSON: " + str(json))
-    # main_logger.warn("Received JSON: " + str(json))
-    # main_logger.warn("Received JSON: " + str(json["URL"]))
+    res = detection.test_old(DetectionData.from_json(json))
 
-    url = json["URL"]
-    screenshot_url = url  # the actual URL to screenshot
+    return res.to_json_str_old()
 
-    # extra json field for evaluation purposes
-    # the hash computed in the DB is the this one
-    if "phishURL" in json:  # TODO only allow this on a testing environment, not prod
-        url = json["phishURL"]
-        main_logger.info(f"Real URL changed to phishURL: {url}\n")
 
-    uuid = json["uuid"]
-    pagetitle = json["pagetitle"]
-    image64 = json["image64"]
+@app.route("/api/v1/url_new", methods=["POST"])
```

Comment:
given that the api endpoint contains version numbers, shall we use them to make a new file for v1 api and switch this one to the main v2?